docs/post-cloner.md

```diff
@@ -0,0 +1,71 @@
+# Post Cloner
+
+This feature adds a button to the post edit screen that allows a user to clone a post with its metadata and terms to a new, identical post.
+
+It is enabled by default but can be disabled via the configuration file:
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"cms": {
+					"cloner": false
+				}
+			}
+		}
+	}
+}
+```
+
+## Actions
+
+**`post_cloner_after_meta_data_copy: (int) $post_id`**
+
+Fires when post meta has been copied to cloned post.
+
+## Filters
+
+**`post_cloner_cloned_parent: (int) $original_post['ID']`**
+
+Allows overriding the parent that the new post should have.
+
+**`post_cloner_cloned_status: (string) 'draft'`**
+
+Allows overriding the status that the new post should have
+
+**`post_cloner_name_append: (string) '-cloned'`**
+
+Allows overriding the string to append onto the end of post name to prevent collisions
+
+**`post_cloner_meta_keys_to_remove: (array) $keys_to_remove`**
```

Comment:
Some of these show the default values but in some cases it's a variable name. Can we take a consistent approach eg. a variable name followed by its default value for each filter?

Also can you add an issue on the upstream repo - I think some of these filters would benefit from passing the `$post_id` as well as a second parameter.